MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/camera/CameraForTest.java

```diff
@@ -189,6 +191,7 @@ public void testGetCameraForGeometryBearing() {
       expectedPosition.bearing, actualPosition.bearing, 0.01f);
   }
 
+  @Ignore("https://github.com/mapbox/mapbox-gl-native-android-internal/issues/69")
```

Comment:
it would be better that we don't use private repo ticket links here in the public repo.

Reply:
Good catch, created a new ticket in this repo to track it.